SchedulerJobs/SchedulerJobs.Services/Mappers/AddNotificationRequestMapper.cs

```diff
@@ -64,13 +65,26 @@ private static NotificationType GetNotificationType(ParticipantResponse particip
                     notificationType = NotificationType.NewHearingReminderRepresentative;
                     break;
                 default:       //JudicialOfficeHolder
+                    if (IsEjudge(participant))
+                    {
+                        notificationType = NotificationType.NewHearingReminderEJUD;
+                    }
+                    else
+                    {
                     notificationType = NotificationType.NewHearingReminderJOH;
```

Comment:
indentation needed

Reply:
fixed. not sure how I lost the indentation :)